### ISSUE TITLE
Publish in the Ubuntu Software Center

### ISSUE THREAD

**Issue by Abbe98** [OpenOrienteering/mapper#603]
Publishing Mapper in Ubuntu Software Center would make Mapper a "first class citizen" in Ubuntu.

Publishing Mapper would make it more accessible for Ubuntu users, support/screenshots/description and other information would show up next to the package.

I could publish it but it would then be under my `albinlarsson` app namespace...

**Comment by dg0yt** [comment]
On a first glance, I don't really understand how to publish an application (not a HTML/QML app) there. And the Ubuntu Software Center is to be replaced with Gnome Software already in 16.04. There seems to be another, cross distribution effort for publish app metadata, called AppStream.

**Comment by Abbe98** [comment]
True, but as far as I know(and the only thing that makes sense) all apps & packages will not need to be re-registered.

Publishing is done through [Ubuntu MyApps](https://developer.ubuntu.com/en/publish/).

**Comment by dg0yt** [comment]
I studied these pages. I don't want to publish a web app or mobile app. I needed to go the footer to find [other forms of submitting apps](https://developer.ubuntu.com/en/publish/other-forms-of-submitting-apps/). Here, they recommend PPAs for "open source apps at no cost", which is no benefit over what we have got from build.opensuse.org. This leaves the "Commercial apps or proprietary apps at no cost" way. We should have everything in place which is needed.

But I have to think twice before I take the burden of handling yet another service. I'm not much in favour of promoting Ubuntu's proprietary services.

(If only this would make progress: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=773110)

**Comment by Abbe98** [comment]
Isn't clear to me what that bug is all about.

It isn't about promoting Ubuntu, but about making the package accessible.

Fully understand if you don't what to maintain another service,  I know the
burden.

**Comment by dg0yt** [comment]
- Mapper in Debian: https://tracker.debian.org/pkg/openorienteering-mapper
- Mapper in Launchpad: https://launchpad.net/ubuntu/+source/openorienteering-mapper
- Screenshot for Ubuntu Software Center: https://screenshots.debian.net/package/openorienteering-mapper

Links added to wiki page [Target Systems](https://github.com/OpenOrienteering/mapper/wiki/Target-systems)

**Comment by Symbian9** [comment]
* Mapper in Launchpad: https://launchpad.net/ubuntu/+source/openorienteering-mapper

Why there no Trusty (LTS)?

**Comment by dg0yt** [comment]
No one made an official package for Trusty (or for Debian experimental) while this distribution was in preparation (before April 2014, i.e. Mapper 0.5.3).

Now Trusty receives only security updates. The current LTS is Xenial, and it is also released as an upgrade for Trusty users since a few days. You should be glad if the official package reaches Xenial-backports. 

And as I said before, I can't promise there will be Mapper 0.7.x packages for Trusty. I really need to ged rid of old workarounds.
